docs/python_docs/python/tutorials/extend/index.rst

```diff
@@ -47,6 +47,12 @@ The following tutorials will help you learn how to customize MXNet.
       How to create new MXNet operators in MXNet's backend using C++.
       An example custom quadratic function op.
 
+   .. card::
+      :title: Using runtime compilation (RTC) to write CUDA kernels in MXNet
+      :link: https://mxnet.apache.org/api/faq/using_rtc
```

Comment:
use `:link: /api/faq/using_rtc` instead as the documentation is versioned.

Reply:
Ok - I was just copy-pasting from the other sections there (like add_op_in_backend). Will update those as well.

Reply:
Hmm, the toctree also has the full links - will `/api/faq/...` approach work there too?

Reply:
Yeah, I tried putting those relative links in the toctree as well and building of Python docs complains with
```
/work/mxnet/docs/python_docs/python/build/tutorials/extend/index.rst:57: WARNING: toctree contains reference to nonexisting document 'api/faq/new_op'                                                              
/work/mxnet/docs/python_docs/python/build/tutorials/extend/index.rst:57: WARNING: toctree contains reference to nonexisting document 'api/faq/add_op_in_backend'                           
/work/mxnet/docs/python_docs/python/build/tutorials/extend/index.rst:57: WARNING: toctree contains reference to nonexisting document 'api/faq/using_rtc'
```
and those entries are not shown in the final website.

Reply:
I'm currently fixing a number of issues in https://github.com/apache/incubator-mxnet/pull/18839. You may get conflict from this

Reply:
Nice :-). I will push the changes to address your other comments then and will get back to the website part once your PR is merged.

Reply:
#18839 is merged, but it did not tackle the issue of toctree - should I leave the links as they are right now @szha?